src/ds/d500/d500-options.h

```diff
@@ -74,4 +75,18 @@ namespace librealsense
         temperature_component _component;
         const char* _description;
     };
+
+    class power_line_freq_option : public uvc_pu_option
+    {
+    public:
+        explicit power_line_freq_option(const std::weak_ptr< uvc_sensor >& ep, rs2_option id,
+            const std::map< float, std::string >& description_per_value);
+
+        virtual option_range get_range() const override
+        {
+            // this range had to be harcoded to avoid collisions with linux patches 
+            // which have been upstreamed for d400 devices
+            return { 0.f /*min*/, 2.f /*max*/, 1.f /*step*/, 0.f /*default*/ };
```

Comment:
Is it possible to make it a lazy when we query the range from PU and only override the min/max?

I think it will be better since each camera can have a different default, here we are forcing it.
Thoughts?

Reply:
sounds great!